CHANGELOG.md

```diff
@@ -24,6 +27,11 @@
   [SimplyDanny](https://github.com/SimplyDanny)
   [#5153](https://github.com/realm/SwiftLint/issues/5153)
 
+* Revert `optional_data_string_conversion` enforcing
+  non-failable conversions of `Data` -> UTF-8 `String`.
```

Comment:
```suggestion
* Revert the part of the `non_optional_string_data_conversion`
  rule that enforces non-failable conversions of `Data` -> UTF-8
  `String`. This is due to the fact that the data to be converted
  can be arbitrary and especially doesn't need to represent a valid
  UTF-8-encoded string.    
```

Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.

Also, this should go into the "Breaking" section.